docs/docusaurus/docs/guides/expectations/creating_custom_expectations/how_to_use_custom_expectations.md

```diff
@@ -24,7 +21,9 @@ This guide will walk you through the process of utilizing Custom Expectations, w
 
 </Prerequisites>
 
-## Steps
+There are slight differences when working with custom expectations or those contributed to GX.
```

Comment:
Maybe a slight nit but when do we use `expectation` vs `Expectation`? Same note for any domain-specific concepts